packages/unified-lint-rule/index.d.ts

```diff
@@ -3,8 +3,13 @@ import type {VFile} from 'vfile'
 import type {Plugin} from 'unified'
 import type {Label, Severity} from './lib/index.js'
 
+export interface RuleMeta {
+  origin: string
+  url?: string
```

Comment:
also including the doc comments here

```suggestion
  /**
   * name of the lint rule
   */
  origin: string
  /**
   * link to documentation
   */
  url?: string
```

Reply:
definitely typed has been moving towards this convention for optional properties
due to https://devblogs.microsoft.com/typescript/announcing-typescript-4-4/#exact-optional-property-types
not sure if we'd want to do that here too

```suggestion
  origin: string
  url?: string | undefined
```

Reply:
I don’t know why anyone would want to use this, but I see no harm in it.